packages/neuron-ui/src/components/History/hooks.ts

```diff
@@ -2,7 +2,7 @@ import { useState, useEffect } from 'react'
 import { updateTransactionList } from 'states/stateProvider/actionCreators/transactions'
 import { listParams, backToTop } from 'utils'
 
-export const useSearch = (search: string = '', walletID: string = '', dispatch: React.Dispatch<any>) => {
+export const useSearch = (search: string, walletID: string, dispatch: React.Dispatch<any>) => {
```

Comment:
Oh, I tried my best to avoid them, but I think these rules are low-risk based on their types. However, I should still mention them in the PR.
Additionally, there is one thing I forgot to mention that needs attention: I made some changes to the logic of `packages/neuron-wallet/src/services/tx/transaction-generator.ts`, which may be risky.

Reply:
I do agree they won't impact the functionalities from the perspective of business logic.

But are they inevitable?

Reply:
This is done to enable the default-param-last rule (https://github.com/nervosnetwork/neuron/pull/2662/files#diff-ae1172860b354ae2d3bb350d788e34c42636fff3bdef3ab8281862f1f50ab59eL134).
If we don't do it this way, we would have to place it in https://github.com/nervosnetwork/neuron/pull/2662/files#diff-ae1172860b354ae2d3bb350d788e34c42636fff3bdef3ab8281862f1f50ab59eR23.